CMakeLists.txt

```diff
@@ -656,7 +712,12 @@ install(DIRECTORY extlibs/ttmath/ttmath
 # Prepare pkg-config file
 
 set(NETWORKIT_LIBNAME networkit)
-execute_process(COMMAND python3 -c "from version import version; print(version, end='')"
+if(WIN32)
+	set(NETWORKIT_CMAKE_PYTHON_INTERPRETER python)
+else()
+	set(NETWORKIT_CMAKE_PYTHON_INTERPRETER python3)
+endif()
+execute_process(COMMAND ${NETWORKIT_CMAKE_PYTHON_INTERPRETER} -c "from version import version; print(version, end='')"
```

Comment:
What about using [the FindPython3 module](https://cmake.org/cmake/help/latest/module/FindPython3.html#module:FindPython3) that you are already using above?

Reply:
Right, maybe that is also a good point to directly switch to use the module on all platforms.